src/lib/encoder.test.ts

```diff
@@ -249,9 +243,8 @@ describe('encoder', () => {
           valueType: 'bytes32',
           // over the max uint256 allowed, does not fit in 32 bytes
           input: toHex(
-            toBN(
-              '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff' +
-                1,
+            toNumber(
+              '0x1ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff',
```

Comment:
Not sure but looks like not the same value.

Before `0xffffffff...ffffffff + 1`

Now: `0x1fffffff...ffffffff`

Looks like the `0x1f` at the beginning is not the same value. Maybe I am misunderstanding something here in this change?

Reply:
The original though was that adding +1 would give it an additional digit at the front, but a hex number that large + 1 seems weird in code. I can clean it up but essentially it was 0x1000000.... and it now 0x1fffff.... but both numbers are one byte longer. The main problem is that plus one would only work as 0xfffffffn + 1n so adding a 1 to a string will add a 1 at the end instead of a '01' making the number invalid. Hmmm.

Reply:
I don't think this ever worked. Look at this
```js
> Buffer.from(('0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff' + 1).slice(2), 'hex')
<Buffer ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff>
```

Reply:
It does need to be 0x01 and not 0x1 otherwise the same nibble at the end will get us
```console
> Buffer.from('0x01ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff'.slice(2), 'hex')
<Buffer 01 ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff>
```